README.md

```diff
@@ -176,31 +175,36 @@ will immediately revoke access for the specified token.
 The Firebase CLI can also be used programmatically as a standard Node module. Each command is exposed as a function that takes an options object and returns a Promise. For example:
 
 ```js
-var client = require('firebase-tools');
-client.list().then(function(data) {
-  console.log(data);
-}).catch(function(err) {
-  // handle error
-});
-
-client.deploy({
-  project: 'myfirebase',
-  token: process.env.FIREBASE_TOKEN,
-  force: true,
-  cwd: '/path/to/project/folder'
-}).then(function() {
-  console.log('Rules have been deployed!')
-}).catch(function(err) {
-  // handle error
-});
+var client = require("firebase-tools");
```

Comment:
Did prettier just pretty some JS inside MD?  That's next level.

Reply:
Yes. Yes it did. I'm not even mad.